app/src/main/java/de/danoeh/antennapod/fragment/InboxFragment.java

```diff
@@ -33,11 +36,16 @@ public class InboxFragment extends EpisodesListFragment {
     private static final String PREF_DO_NOT_PROMPT_REMOVE_ALL_FROM_INBOX = "prefDoNotPromptRemovalAllFromInbox";
     private SharedPreferences prefs;
 
+    private DateSortType sortType = UserPreferences.getInboxSortOrder();
```

Comment:
Instead of a new DateSortType, I would simply use the existing `SortOrder.DATE_OLD_NEW` and `SortOrder.DATE_NEW_OLD` enums.